### PR TITLE
Add Logos and links to footer

### LINKED ISSUE #49: Add Partner logos/links (as footer if an option)

### LINKED ISSUE #63: Add social links to footer?

### LINKED ISSUE #64: Add Copyright Information in footer

### PR DESCRIPTION
**Pull request by rbeucher**
I Have added the partners logos and links to their website.
I have also added the CC BY logos to the footer.
I have added the social links to our Linkedin, Twitter and Slack

This should:

close #63 
close #64 
close #49

### CODE REVIEW

**Comment by rbeucher** [comment]
Argh... Logos are not showing. Will fix that on Monday...

File: overrides/partials/copyright.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{#-
+  This file was automatically generated - do not edit
+-#}
+<div class="md-copyright">
+  {% if config.copyright %}
+  <div class="md-copyright__highlight">
+    <div>
+      <a rel="license" href="https://creativecommons.org/licenses/by/4.0/" title="Creative Commons Attribution 4.0 International license">
+      <i class="icon-cc"></i>
+      <i class="icon-cc-by"></i>
```
Comment (ccarouge) [inline overrides/partials/copyright.html:10]: This one should be CC0 not CC-BY

File: mkdocs-community.yml [hunk 2]
```diff
@@ -84,3 +85,24 @@ nav:
...
+      link: https://twitter.com/ACCESS_NRI
+      name: ACCESS-NRI on Twitter
+    - icon: fontawesome/brands/linkedin
+      link: https://www.linkedin.com/in/access-nri
+      name: ACCESS-NRI on Linkedin
+    - icon: fontawesome/brands/slack
+      link: https://access-nri.slack.com/
+      name: Join ACCESS-NRI Slack Channel
+
+copyright: CC-BY-4.0
```
Comment (ccarouge) [inline mkdocs-community.yml:105]:
```suggestion
copyright: CC0
```

File: access-nri/overrides/partials/footer.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          </a>
+          <a href="https://www.unimelb.edu.au" target="_blank" rel="noopener" title="The University of Melbourne">
+          <img src="assets/university-of-melbourne_logo.png" alt="logo" style="width: auto; height: 2.1rem; padding: 0.3rem;">
+          </a>
+          <a href="https://www.unsw.edu.au" target="_blank" rel="noopener" title="The University of New South Wales">
+          <img src="assets/university-of-nsw_logo.png" alt="logo" style="width: auto; height: 2.1rem; padding: 0.3rem;">
+          </a>
+          <a href="https://www.utas.edu.au" target="_blank" rel="noopener" title="The University of Tasmania">
+          <img src="assets/university-of-tasmania_logo.svg" alt="logos" style="width: auto; height: 2.1rem; padding: 0.3rem;">
+          </a>
```
Comment (ccarouge) [inline access-nri/overrides/partials/footer.html:31]: Should we add ourselves? For the moment, there is no obvious link to ACCESS-NRI website.

File: overrides/partials/footer.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+          </a>
+          <a href="https://www.unimelb.edu.au" target="_blank" rel="noopener" title="The University of Melbourne">
+          <img src="assets/university-of-melbourne_logo.png" alt="logo" style="width: auto; height: 2.1rem; padding: 0.3rem;">
+          </a>
+          <a href="https://www.unsw.edu.au" target="_blank" rel="noopener" title="The University of New South Wales">
+          <img src="assets/university-of-nsw_logo.png" alt="logo" style="width: auto; height: 2.1rem; padding: 0.3rem;">
+          </a>
+          <a href="https://www.utas.edu.au" target="_blank" rel="noopener" title="The University of Tasmania">
+          <img src="assets/university-of-tasmania_logo.svg" alt="logos" style="width: auto; height: 2.1rem; padding: 0.3rem;">
+          </a>
```
Comment (ccarouge) [inline overrides/partials/footer.html:31]: Should we add ourselves? For the moment, there is no obvious link to ACCESS-NRI website.

**Review by ccarouge — CHANGES_REQUESTED** [review]
It looks great. A few minor changes. It would be great if you could rebase on `main` before accepting the PR since I've merged the home and license stuff.

File: access-nri/overrides/partials/footer.html (hunk 3 above)
Comment (rbeucher) [reply access-nri/overrides/partials/footer.html]: Not sure...

**Pull request merged**